queries/num.py

```diff
@@ -365,8 +415,25 @@ def year_to_text(year: int, *, after_christ: bool = False) -> str:
     return " ".join(text) + suffix
 
 
-DeclensionMapping = Mapping[str, Mapping[str, Mapping[str, str]]]
-_FYRSTUR_STRONG_DECL: DeclensionMapping = {
+def years_to_text(
+    s: str, *, regex: str = r"(?<=\b)\d\d\d\d(?=\b)", after_christ: bool = False
+) -> str:
+    """
+    Converts years of the form '1994', '2021'
+    (or matching regex if provided)
+    in string to Icelandic text.
+    Can also be supplied with custom pattern to match years with shorter length than 4.
+    """
+
+    def convert(m):
+        match = m.group(0)
+        n = int(match.strip(" "))
```

Comment:
Do you need .strip(), and then why not straight .strip() instead of .strip(" ")?

Reply:
Whoops, probably don't need it.